schema/question.query.graphql

```diff
@@ -0,0 +1,15 @@
+query getQuestion(
```

Comment:
Unless there's a change to bring in something that'll import the query where it's used, we can delete it.